lib/screens/repositories_screen.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+
+import '../widgets/drop_down.dart';
+
+class RepositoriesScreen extends StatefulWidget {
+  const RepositoriesScreen({super.key});
+
+  @override
+  State<RepositoriesScreen> createState() => _RepositoriesScreenState();
+}
+
+class _RepositoriesScreenState extends State<RepositoriesScreen> {
+  final List<String> _languages = [
+    "Dart",
+    "Java",
+    "JavaScript",
+    "Kotlin",
+    "PHP",
+    "Python",
+    "Ruby",
+    "Swift",
+    "TypeScript",
+  ];
+  String? _selectedLanguage;
+  List<String> repositories = [];
+
+  void _didSelectItem(String item) {
```

Comment:
A better naming of this function would be:
```suggestion
  void _didSelect({required String item}) {
```

Usage: _didSelect(item: "")

Reply:
done